### ISSUE TITLE
Mirror Images threat issue

### ISSUE THREAD

**Issue by xethra89** [SunwellTracker/issues#1502]
Decription: Mirror images will increase threat of the mage by an impossible amount.

How it works: Mage uses mirror images, when mirror images disappates an unknown amount of threat is transferred to the mage who casted mirror images.

https://clips.twitch.tv/HelpfulBusyTurtlePanicVis

How it should work: Mirror images should transfer no threat or if it does transfer threat, it should not be in the billions of amounts.

Source (you should point out proofs of your report, please give us some source): Common sense?

**Comment by Badrng** [comment]
Using mirror images reduces the mage's threat by a massive amount (around 30 million if I remember correctly) but only for as long as the spell is active (that is 30 seconds). During that time you keep generating threat, it's just it's massively reduced. Once those 30 seconds pass, the effect that reduces your threat dissipates and you regain all that threat (which is probably the "unknown amount" you're referring to). Also, since you've been generating threat as normal during that time, you'll find that you have even more of it than before you used thje images. AFAIK the spell works as intended, at least the threat reduction part does.

**Comment by xethra89** [comment]
@Badrng https://media.discordapp.net/attachments/403602613297348609/455348957615357952/asda.jpg?width=1202&height=677

4 million threat is not even close to possible to generate within 15 seconds, not even for an ICC geared tank.

**Comment by Cwiercpasiec** [comment]
Yup, can confirm it, sometimes mirror images give about 4 milion threat after disappearing.

**Comment by Badrng** [comment]
I've never noticed this happen, my guess is that the mage has exceeded the tank's aggro during mirror images and once the effect wore off the threat meter went haywire. But perhaps I'm wrong, haven't played my mage for a while.

**Comment by gabbat** [comment]
happend to me as well once, can confirm something is off with mirror images:
used invisibility to drop all threat, like 10seconds later images dissapeared and I pulled aggro instantly which is literally impossible after using invisibility
